test/utils/image/manifest.go

```diff
@@ -43,6 +43,10 @@ func (i *ImageConfig) SetName(name string) {
 	i.name = name
 }
 
+func (i *ImageConfig) GetName() string {
```

Comment:
Please remove this function or rename it to `Name()`. Getter functions in Go are not common and if you decide to use them, they should not have the "Get" prefix.

Reply:
Addressed (though perhaps Name() should've been better put before all setters). Thanks Bobby!